src/main/java/ch/uzh/ifi/hase/soprafs22/controller/GameController.java

```diff
@@ -98,174 +98,174 @@ public void createGame(@RequestHeader(value = "authorization", required = false)
 
-      userService.checkSpecificAccess(token, id);
+    userService.checkSpecificAccess(token, id);
 
-      // Get 'random' game
-      Game game = gameService.getGameFromRandomUser(id);
+    // Get 'random' game
+    Game game = gameService.getGameFromRandomUser(id);
 
-      // return it
-      CardAndGameIdGetDTO cardAndGameIdGetDTO = new CardAndGameIdGetDTO();
+    // return it
+    CardAndGameIdGetDTO cardAndGameIdGetDTO = new CardAndGameIdGetDTO();
 
-      cardAndGameIdGetDTO.setBlackCard(game.getBlackCard());
-      cardAndGameIdGetDTO.setGameId(game.getId());
+    cardAndGameIdGetDTO.setBlackCard(game.getBlackCard());
+    cardAndGameIdGetDTO.setGameId(game.getId());
 
-      return cardAndGameIdGetDTO;
-    }
+    return cardAndGameIdGetDTO;
+  }
 
-    /**
-    * User gets his white Cards
-     */
-    @GetMapping("users/{userId}/games/whiteCards")
-    @ResponseStatus(HttpStatus.OK)
-    @ResponseBody
-    public List<CardGetDTO> getWhiteCardsFromUser(@RequestHeader(value = "authorization", required = false) String token,
-                                                @PathVariable(value = "userId") Long id) {
-      // check token
-      userService.checkSpecificAccess(token, id); // throws 401, 404
-
-      // get white cards of user
-      List <WhiteCard> cards = userService.getWhiteCards(id);
-
-      // return them
-      List<CardGetDTO> cardGetDTO= new ArrayList<>();
-      for (WhiteCard card : cards){
-          cardGetDTO.add(DTOMapper.INSTANCE.convertEntityToCardGetDTO(card));
-      }
-      return cardGetDTO;
-    }
+  /**
+  * User gets his white Cards
+    */
+  @GetMapping("users/{userId}/games/whiteCards")
+  @ResponseStatus(HttpStatus.OK)
+  @ResponseBody
+  public List<CardGetDTO> getWhiteCardsFromUser(@RequestHeader(value = "authorization", required = false) String token,
+                                              @PathVariable(value = "userId") Long id) {
+    // check token
+    userService.checkSpecificAccess(token, id); // throws 401, 404
 
-    /**
-     * get a game with all plays
-     */
-    @GetMapping("/users/{userId}/games/vote")
-    @ResponseStatus(HttpStatus.OK)
-    @ResponseBody
-    public GameGetDTO getGame(@RequestHeader(value = "authorization", required = false) String token,
-                        @PathVariable(value = "userId") Long id) {
-
-      userService.checkSpecificAccess(token, id); // throws 401, 404
-
-      User user = userService.getUserById(id);
-      // get one game (first get the old ones)
-      Game game = gameService.getGame(user.getActiveGame(), user.getPastGames());
-      return DTOMapper.INSTANCE.convertEntityToGameGetDTO(game);
+    // get white cards of user
+    List <WhiteCard> cards = userService.getWhiteCards(id);
+
+    // return them
+    List<CardGetDTO> cardGetDTO= new ArrayList<>();
+    for (WhiteCard card : cards){
+        cardGetDTO.add(DTOMapper.INSTANCE.convertEntityToCardGetDTO(card));
     }
+    return cardGetDTO;
+  }
```

Comment:
Did you do anything here? This makes it somewhat difficult to see what you actually did, so if you clean up code, better use some temporary clean-up branch

Reply:
no, just a bit of reformating